pentaho-gwt-widgets/src/org/pentaho/mantle/client/dialogs/scheduling/RecurrenceEditor.java

```diff
@@ -369,12 +371,13 @@ private void createTemporalMap() {
     public SimpleRecurrencePanel( String strLabel ) {
 
       HorizontalPanel hp = new HorizontalPanel();
+      hp.getElement().setId( RECUR_PATTERN_HP );
       Label l = new Label( Messages.getString( "schedule.every" ) );
       l.setStyleName( "startLabel" ); //$NON-NLS-1$
       hp.add( l );
 
-      valueTb.setWidth( "3em" ); //$NON-NLS-1$
```

Comment:
yes it does, but I set the width at 3em in crystal and onyx in their respective css files rather than inline